README.md

```diff
@@ -78,6 +78,117 @@ createAction('ADD_TODO')('Use Redux');
 
 `metaCreator` is an optional function that creates metadata for the payload. It receives the same arguments as the payload creator, but its result becomes the meta field of the resulting action. If `metaCreator` is undefined or not a function, the meta field is omitted.
 
+### `createActions(?actionMap, ?arrayDefTypes, ?optionMap)`
```

Comment:
I think a better name for `arrayDefTypes` would be `identityActionTypes`, since that captures the fact that

- they're action types
- they use the identity function for the payload creator

If you used [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) (better, IMO), you also wouldn't have to encode the type in the variable name, and it would be by default an array.